### PR TITLE
Sample onboarding

### PR DESCRIPTION
**Pull request by hferrone**

### CODE REVIEW

File: HL2Collab/README.md [hunk 1]
```diff
@@ -1,21 +1,61 @@
-# HL2Collab
-## _Multiplayer Networking on the HoloLens_
 ---
-# Instructions
+page_type: sample
+name: HoloLens Multiplayer Networking in Unity
```
Comment (fieldsJacksonG) [inline HL2Collab/README.md:3]: Unreal

File: HL2Collab/README.md [hunk 2]
```diff
@@ -1,21 +1,61 @@
...
+description: This sample shows how to network multiple HoloLens 2 devices using the built-in Unreal Engine networking system.
+languages:
+- cpp
+products:
+- windows-mixed-reality
+- hololens
+---
+
+# HL2Collab - HoloLens Multiplayer Networking in Unity
```
Comment (fieldsJacksonG) [inline HL2Collab/README.md:12]: Unreal

File: HL2Collab/README.md [hunk 3]
```diff
@@ -1,21 +1,61 @@
...
 
-- Open the *Level* blueprint in MainMenuMap. This contains instructions as well.
-- Set the **IP_TextField** variable to your computer's IP Adress
-- Create a Desktop Build. This will be the host
-- Create HL2 or more Desktop builds to act as clients
-- Run the Desktop version, and press "H" to begin hosting the map.
-- Run the HL2 clients, and "right pinch" to join the session to the host on the IP Address provided earlier
-- (Optional) Run any other Desktop executables, and press "J" to join
+<!-- Need these -->
+Needs OpenXR - need input from Sajid, Jackson, Summer
```
Comment (fieldsJacksonG) [inline HL2Collab/README.md:38]: In the Microsoft OpenXR Plugin GitHub, we have a section on adding the plugin to your project from the Unreal Marketplace:
https://github.com/microsoft/Microsoft-OpenXR-Unreal#install-the-plugin-from-the-unreal-marketplace

Once the Microsoft OpenXR Plugin is enabled, all other OpenXR prerequisite plugins will also be enabled, then setup for a HoloLens app resumes as normal

**Review by fieldsJacksonG — CHANGES_REQUESTED** [review]

File: HoloLens2Example/README.md [hunk 4]
```diff
@@ -9,18 +21,42 @@ Supported Unreal versions | Supported device
...
+| `HoloLens2Example.uproject` | Unreal Engine project file. |
+| `README.md` | This README file. |
+
+## Prerequisites
+
+<!-- Need these -->
+
+# Setup
 
 Make sure you have Unreal Engine 4.24 or later installed on your computer.
```
Comment (sw5813) [inline HoloLens2Example/README.md:42]: 4.26

File: HoloLens2Example/README.md [hunk 5]
```diff
@@ -9,18 +21,42 @@ Supported Unreal versions | Supported device
...
 * If you do not have a HoloLens 2 device, first follow the [App Packaging Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/HowTo/PackageApp/index.html) and then follow the [Emulator Quickstart Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/QuickStartEmulator/index.html) to deploy the app to the [HoloLens 2 Emulator](https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-hololens-emulator)
 * If you have a HoloLens 2 device and want to run the app on device, first follow the [App Packaging Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/HowTo/PackageApp/index.html) and then follow the [Device Quickstart Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/QuickStartDevice/index.html) to deploy the app to your HoloLens 2
 * If you have a HoloLens 2 device and want to stream the app to device, follow the [Streaming Quickstart Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/QuickStartStreaming/index.html) to stream the app from the Unreal editor to your HoloLens 2
 
-## How to use the app
+## Running the sample
```
Comment (sw5813) [inline HoloLens2Example/README.md:47]: I actually think "How to use the app" is a better descriptor of the below section, as it's instructing readers how to interact with the app after it's deployed and running on device or emulator. "Running the sample" would make more sense if the instructions were teaching the user how to get the app running in the first place. I don't think there's really a difference between this and "Setup".

**Review by sw5813 — CHANGES_REQUESTED** [review]
Thanks for fleshing out these docs, Harrison!

File: HoloLens2Example/README.md (hunk 5 above)
Comment (hferrone) [reply HoloLens2Example/README.md:47]: Agreed, but these sections are how the samples directory wants the README's structured: https://review.docs.microsoft.com/en-us/help/contribute/samples/concepts/readme-template?branch=master

**Pull request merged**